Makefile-os

```diff
@@ -46,7 +43,7 @@ rootshell: ## connect to a running addons-server docker shell with root user
 
 .PHONY: create_env_file
 create_env_file:
-	echo "UID=${UID}\nGID=${GID}" > .env
+	echo "HOST_UID=${HOST_UID}" > .env
```

Comment:
Is there a drawback to leaving `UID` in there, to keep compatibility with older `docker-compose.yml` ? I'm thinking it could help if we go back to an earlier commit to try something out (maybe as part of a `bisect`) and we don't want to re-initialize the env file every time.

Reply:
I tested and it works... but I'd be wary of that. Setting a value to UID in the first place was kind of sketchy. the default value should be the UID of the current user, but by setting it we are then changing that. by keeping it in, we run the risk of that kind of a value collision in the future.

How long should we support it? I might prefer the set of error cases that you described which should reduce over time whereas keeping it keeps open a set of potential new error cases with no real end date.

Reply:
Ok, let's not do that, we'll see if we run into those kind of issues.

Reply:
Also just found out, we'd have to map it in the docker-compose.yml too. I'm seeing a pattern here that seems to stem from the idea that we have long running local environments that need to be supported or maintained over time.

I don't really see how you can have that in a codebase that is changing. When you run in production you build a new env and blow away the old one. Local should work like that too. If you are checking out a super old branch, and you don't recreate the local environment based on the state of the app at that time, you aren't really reproducing the same environment but just loading old code on to your current env.

I'm not sure this is a solvable problem and I would advocate we move to "blow it away" and add a feature for backing up specific data you want to hold onto across builds.